liblzma-config.sh

```diff
@@ -10,6 +10,14 @@ cd "$TARGET_DIR"
 tar xvjf "$SRC_TARBALL" >node_liblzma_config.log 2>&1 
 
 export CFLAGS="-fPIC $CFLAGS"
+
+# Fix build on Apple Silicon
+if [[ ("$OSTYPE" == "darwin"*) && ($(uname -m) == "arm64") ]]; then
+	XZ_SRC_DIR=$(ls | grep xz-*)
+	sed -i.bak 's/\tnone)/\tarm64-*)\n\t\tbasic_machine=$(echo $basic_machine | sed "s\/arm64\/aarch64\/")\n\t\t;;\n\t\tnone)/g' $XZ_SRC_DIR/build-aux/config.sub
+	rm $XZ_SRC_DIR/build-aux/config.sub.bak
```

Comment:
Could remove this line and the `.bak` on line 17 (leave `-i` for in-place replace)

Reply:
DONE!